english/chess/chess.md

```diff
@@ -1,3 +1,6 @@
+If you chose this option because you misread it as "Cheese Puzzle" [don't be sad](english/chess/CheesePuzzle.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.